logiclearner/settings_production.py

```diff
@@ -16,6 +16,16 @@
         #       s3static=False,
     ))
 
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
```

Comment:
Postgis won't be needed with Logic Learner. We can use the default `'django.db.backends.postgresql_psycopg2',`